ROADMAP.md

```diff
@@ -0,0 +1,62 @@
+# Parsec Project Roadmap
+
+## Notes
+
+- This roadmap document is a community resource. It is intended to help the project maintenance team
+   to share and evolve a common understanding of the project roadmap with all contributors and
+   stakeholders. It is for information purposes only, and it should be viewed as being both flexible
+   and approximate.
+- The release process for the Parsec project is documented
+   [here](https://parallaxsecond.github.io/parsec-book/contributing/release_process.html).
+- This roadmap document is focused on major/headline features and fixes. For full details and
+   changelogs for historical releases, please refer to the [releases
+   page](https://github.com/parallaxsecond/parsec/releases).
+- Full details of the current cryptographic capabilities of Parsec on the various platforms are
+   available
+   [here](https://parallaxsecond.github.io/parsec-book/parsec_client/operations/service_api_coverage.html).
+- The section on previous releases does not include any that were made before there was a defined
+   process and cadence for Parsec releases.
+
+# Previous Releases
+
+## 0.8.1 (September 2021)
+
+- Full [release page](https://github.com/parallaxsecond/parsec/releases/tag/0.8.1).
+- Support for [SPIFFE](https://spiffe.io)-based authentication to API operations. (Note: Rust client
+   library only).
+- Support for ECC keys in PKCS#11 devices.
+- Support for secure elements using [CryptoAuthLib](https://github.com/MicrochipTech/cryptoauthlib).
+- Fixes persistence of keys across reboot in TPM devices.
+- TPM provider can be enabled/disabled dynamically based on detection of a TPM device.
+
+# Future Releases
```

Comment:
+1

Reply:
and reverse sorting for the old releases

Reply:
Yes, that makes sense - in fact I'm not even convinced we need the previous releases section at all, because it's essentially paraphrasing what's on the GH releases page. I will move it to the end, and then we can take a decision later about maybe removing it completely. The future roadmap is what really matters.